chromadb/db/__init__.py

```diff
@@ -102,26 +106,28 @@ def delete(
         collection_uuid: Optional[UUID] = None,
         ids: Optional[IDs] = None,
         where_document: WhereDocument = {},
-    ) -> List:
+    ) -> List[str]:
```

Comment:
This is not correct? Both the duckdb and clickhouse classes return UUIDs?

https://github.com/chroma-core/chroma/blob/d88860bbc50f6b25eac568dbefee380440c3f27d/chromadb/db/duckdb.py#L176